src/reporters/build_kite_cli.ts

```diff
@@ -70,42 +62,44 @@ export default class BuildKiteCLIReporter extends BaseReporter {
     const { failed, succeeded, skipped } = this.metrics;
     const total = failed + succeeded + skipped;
     if (total === 0 && error) {
-      this.write(renderError(error));
+      this.write(renderError(serializeError(error)));
```

Comment:
there is a indentation issue here. I would just do the same here by calling `super.onJourneyEnd` and then add the `Took duration message`